ddtrace/_trace/sampler.py

```diff
@@ -101,7 +102,7 @@ def __init__(
         else:
             self.rules: List[SamplingRule] = rules or []
         # Set Agent based samplers
-        self._by_service_samplers: Dict[str, RateSampler] = {}
+        self._agent_sampling_rules = agent_sampling_rules or {}
```

Comment:
`_agent_based_samplers` might be better here, since the use of the term "rules" conflicts with the way it's used elsewhere in the library as opposed to "sampler".